src/app/static/src/app/components/plots/table/Table.vue

```diff
@@ -2,23 +2,34 @@
     <div>
         <br>
         <div v-if="filteredData.length > 0">
-            <table class="table">
-                <div>
-                    <tr>
-                        <th v-translate="'area'"></th>
-                        <th v-for="f in filtersToDisplay" v-translate="f.label"></th>
-                        <th v-for="i in indicators">{{ i.name }}</th>
-                    </tr>
-                    <tr v-for="row in filteredData">
-                        <td>
-                          <div>{{ row.areaLabel }}</div>
-                          <div class="small">{{ row.areaHierarchy }}</div>
-                        </td>
-                        <td v-for="f in filtersToDisplay">{{ row.filterLabels[f.id] }}</td>
-                        <td v-for="i in indicators">{{ row.indicatorValues[i.indicator] }}</td>
-                    </tr>
-                </div>
-            </table>
+            <b-form-group
+              class="mb-0"
+            >
+              <b-input-group size="sm">
+                <b-form-input
+                  v-model="filter"
+                  type="search"
+                  id="filterInput"
+                  placeholder="Type to Search"
+                ></b-form-input>
+                <b-input-group-append>
+                  <b-button :disabled="!filter" @click="filter = ''">Clear</b-button>
```

Comment:
I'm not sure you really need this Clear button, as you seem to get an 'x' which appears in the field when there's any filter text, and which has the same effect.
If we keep it, it'll need a translation too!

Reply:
I think having a clear button is nice to have as it signals to the user what the filter tool is/does.